oms-monitor/docker/logstash/pipeline/logstash.conf

```diff
@@ -81,6 +82,7 @@ filter {
     grok {
       match => { "message" => ["%{IPORHOST:[nginx][access][remote_ip]} - %{DATA:[nginx][access][user_name]} \[%{HTTPDATE:[nginx][access][time]}\] \"%{WORD:[nginx][access][method]} %{DATA:[nginx][access][url]} HTTP/%{NUMBER:[nginx][access][http_version]}\" %{NUMBER:[nginx][access][response_code]} %{NUMBER:[nginx][access][body_sent][bytes]} \"%{DATA:[nginx][access][referrer]}\" \"%{DATA:[nginx][access][agent]}\""] }
       remove_field => "message"
+      add_tag => ["nginx_access-log"]
```

Comment:
inconsistency: `_` here and `-` on line 29

Reply:
Ah yes, forgot to update this one. Thanks for noticing!

Reply:
fixed